qcodes/instrument_drivers/QuTech/M2j.py

```diff
@@ -0,0 +1,83 @@
+from qcodes.instrument.base import Instrument
+from qcodes.utils.validators import Numbers
+import numpy as np
+
+try:
+    from spirack import M2j_module
+except ImportError:
+    raise ImportError(('The M2j_module class could not be found. '
+                       'Try installing it using pip install spirack'))
+
+
+class M2j(Instrument):
+
+    def __init__(self, name, spi_rack, module, **kwargs):
+        """
+        Qcodes driver for the M2j RF amplifier SPI-rack module.
+
+        Args:
+            name (str): name of the instrument.
+
+            spi_rack (SPI_rack): instance of the SPI_rack class as defined in
+                the spirack package. This class manages communication with the
+                individual modules.
+
+            module (int): module number as set on the hardware.
+
+        The gain and RF_level can only be set to the device, but not read from the device.
+
+        Parameters:
+            gain: Amplifier gain in dB, range 32 to 55 dB
```

Comment:
I recommend that you add these descriptions as `docstring` argument of the `add_parameter` method.